ambry-store/src/test/java/com.github.ambry.store/IndexSegmentTest.java

```diff
@@ -35,18 +38,26 @@
 import java.util.concurrent.atomic.AtomicLong;
 import org.junit.After;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
 import static org.junit.Assert.*;
 
 
 /**
  * Tests for {@link IndexSegment}.s
  */
+@RunWith(Parameterized.class)
 public class IndexSegmentTest {
   private static final int CUSTOM_ID_SIZE = 10;
   private static final int KEY_SIZE = new MockId(UtilsTest.getRandomString(CUSTOM_ID_SIZE)).sizeInBytes();
-  private static final int VALUE_SIZE = new IndexValue(0, new Offset("", 0)).getBytes().capacity();
+  private static final int VALUE_SIZE_V0 =
```

Comment:
any reason not to use `INDEX_VALUE_SIZE_IN_BYTES_V0` and `INDEX_VALUE_SIZE_IN_BYTES_V1`?